### DISCUSSION TITLE
Cannot create the first zettel

### DISCUSSION THREAD

**Discussion by hadronized** [srid/neuron#656] [General]
**Describe the bug**
I cannot create the first note because of an error:

```
$ neuron new -e
   Plugins enabled: links, tags, neuronignore, uptree, feed
   Building graph (0 notes) ...
/home/phaazon/zettelkasten/89ea9c58.md
neuron: bash: executeFile: does not exist (No such file or directory)
```

**To Reproduce**
1. `mkdir ~/zettelkasten`
2. `cd ~/zettelkasten`
3. `touch neuron.dhall`
4. `neuron new -e`

**Expected behavior**
Should open the editor to enter the note.

**Desktop (please complete the following information):**
N/A

**Additional context**
- `uname -a`: `Linux orchid 5.12.15-arch1-1 #1 SMP PREEMPT Wed, 07 Jul 2021 23:35:29 +0000 x86_64 GNU/Linux`

**Comment by srid** [comment]
Your `EDITOR` environment variable is not a valid executable. Try,

```
EDITOR=nano neuron new -e
````

And tweak things from there.

**Reply by hadronized** [reply]
It’s set to `nvim`, which is valid. Also, closing the issue while you have a nasty error message doesn’t seem the right thing to do in terms of UX. At least fix the error message and provide users a better description of the problem, which, in my case, is unlikely to be the `EDITOR`.
